ui-modules/blueprint-composer/app/views/main/graphical/graphical.state.html

```diff
@@ -19,129 +19,258 @@
+                      </span>
+
+                        (<a class="hand" ng-click="errorsPane.focus = (errorsPane.focus == 'errors:'+itemK ? null : 'errors:'+itemK)"
+                            ng-class="{active: errorsPane.focus == 'errors:'+itemK }">
+                            <ng-pluralize count="itemV.issues.length"
+                                          when="{'one': '{} entity', 'other': '{} entities'}" />
+                        </a>
+                        <span ng-if="vm.size(itemV.quickFixes)">;
+                          <a class="hand" ng-click="errorsPane.focus = (errorsPane.focus == 'fixes:'+itemK ? null : 'fixes:'+itemK)"
+                             ng-class="{active: errorsPane.focus == 'fixes:'+itemK }">
+                             <ng-pluralize count="vm.size(itemV.quickFixes)"
+                                           when="{'one': '{} quick fix', 'other': '{} quick fixes'}" />
+                             available
+                          </a>
+                        </span>)
+
+                        <div class="error-line-sub" ng-if="errorsPane.focus == 'errors:'+itemK">
+                            <div ng-repeat="issue in itemV.issues" class="error-line-sub-line">
+                                <div class="error-line-marker">
+                                    <i class="fa fa-fw fa-circle"></i>
+                                </div>
+                                <div class="error-line-text">
+                                    {{ vm.entitySummary(issue.issue.entity) }}
+                                    <!-- could offer the issue-specific quick fixes; but clearer to navigate to entity and do there -->
+                                </div>
+                                <div class="error-line-action">
+                                    <a class="hand" ui-sref="main.graphical.edit.entity({entityId: issue.issue.entity._id})">
+                                        <i class="fa fa-fw fa-external-link"></i>
+                                    </a>
+                                </div>
+                            </div>
+                        </div>
+
+                        <div class="error-line-sub" ng-if="errorsPane.focus == 'fixes:'+itemK">
+                            <div ng-repeat="fix in itemV.quickFixes" class="error-line-sub-line">
+                                <div class="error-line-marker">
+                                    <i class="fa fa-fw fa-magic"></i>
+                                </div>
+                                <div class="error-line-text">
+                                    {{ fix.text }}
+                                    <a class="hand btn btn-xs btn-primary" style="float: right;" ng-click="vm.applyQuickFix(fix)"
+                                       ng-attr-title="{{ fix.tooltip }}"
+                                    >Apply
+                                        (<ng-pluralize count="vm.size(fix.issues)"
```

Comment:
Typo: redundant parentheses at the beginning.

Reply:
Or make parentheses at the end visible. It is not displayed.

Reply:
@algairim It is at the end of the `ng-pluralize` directive, on line 197

Reply:
<img width="187" alt="Screenshot 2021-07-09 at 09 13 46" src="https://user-images.githubusercontent.com/81319331/125046483-f8920480-e095-11eb-81f8-d91f03c78e39.png">
parentheses  at 197 is not displayed.

Reply:
Urgh, forgot that AngularJS is picky with self-closing tag. It's fixed now

Reply:
Now it works, thanks @tbouron